Mage/src/main/java/mage/abilities/keyword/ExertAbility.java

```diff
@@ -53,6 +57,9 @@ public ExertAbility(BecomesExertSourceTriggeredAbility ability, boolean exertOnl
         ruleText += " <i>(An exerted creature won't untap during your next untap step.)</i>";
         if (exertOnlyOncePerTurn) {
             getWatchers().add(new ExertedThisTurnWatcher());
+
+            addHint(new ConditionHint(NotExertedThisTurnCondition.instance, "{this} hasn't been exerted this turn",
+                    null, "{this} has been exerted this turn", null, true));
```

Comment:
what's the rationale for a hint on exerting, rather than the not untapping? / isn't it already covered by DontUntapInControllersNextUntapStepTargetEffect ?

Reply:
This was targeted towards Combat Celebrwnt, which says “ If Combat Celebrant hasn’t been exerted this turn, you may exert it as it attacks”… I was going to add it specifically for combat celebrant, but it seemed like a piece of text that could come up again and we’d want to support

Reply:
Makes sense, thanks for the explanation